main.go

```diff
@@ -49,6 +49,31 @@ var githubRepoToGitLabProject = map[string]string{
 	"saas": "Northern.tech/MenderSaaS/saas",
 }
 
+var enterpriseRepositories = []string{
+	// backend
+	"deployments-enterprise",
+	"inventory-enterprise",
+	"tenantadm",
+	"useradm-enterprise",
+	"workflows-enterprise",
+	"mender-conductor", // OS in mendersoftware
+	"mender-conductor-enterprise",
+	"mender-helm",
+	// client
+	"mender-binary-delta",
+	"mender-demo-artifact", // OS in mendersoftware
+	// saas
+	"saas",
+	"saas-tools",
+	"sre-tools",
+	// docs
+	"mender-docs",
+	"mender-docs-site",
+	// qa
+	"mender-qa",              // OS in mendersoftware
+	"mender-test-containers", // OS in mendersoftware
+}
```

Comment:
I don't understand the selection of repositories in this list. Why are some, but not all of the OS repositories listed?

Reply:
We need to sync the project that are private in GitLab, regardless of their status in GitHub. Only some OS repositories have private projects in GitLab.

I think that except for `mender-qa`, we could open the other projects in GitLab.

Reply:
Yes, fully agree, open as many as possible. Just be careful that the repo is not exposing any credentials.

Please add a comment about the Github/Gitlab difference, since this was not obvious from the list.

Reply:
Fixed now.